dask/array/tests/test_gufunc.py

```diff
@@ -146,7 +146,7 @@ def stats(x):
 @pytest.mark.parametrize("vectorize", [False, True])
 def test_apply_gufunc_output_dtypes_string_many_outputs(vectorize):
     def stats(x):
-        return np.mean(x, axis=-1), np.std(x, axis=-1)
+        return np.mean(x, axis=-1), np.min(x, axis=-1)
```

Comment:
This acts as a regression test against the example case in #7668 (`min`/`max` is required to trigger that codepath).

Reply:
I added `std` back.